monix-eval/shared/src/main/scala/monix/eval/CoevalLift.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2014-2019 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.eval
+
+import cats.Eval
+import cats.effect._
+
+import scala.annotation.implicitNotFound
+
+/**
+  * A lawless type class that specifies conversions from `Coeval`
+  * to similar data types (i.e. pure, synchronous).
+  */
+@implicitNotFound("""Cannot find implicit value for CoevalLift[${F}].
+Building this implicit value might depend on having an implicit
+s.c.ExecutionContext in scope, a Scheduler or some equivalent type.""")
```

Comment:
Is there any type in the ecosystem that needs EC for a `Sync` instance? I'm afraid that might be more misleading than helpful.

Reply:
No, just a left-over